package.json

```diff
@@ -21,6 +21,7 @@
   "dependencies": {
     "bluebird": "^2.9.34",
     "got": "^4.1.1",
-    "lodash": "^3.10.1"
+    "lodash": "^3.10.1",
+    "sleep": "^3.0.1"
```

Comment:
`sleep` is only used in the tests, it's a development dependency, it should be in a `devDependencies` block:

```json
  "devDependencies": {
      "sleep": "^3.0.1"
  }
```
For the next times, when you want to install a devDependency use `npm install --save-dev myLib`